package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "channelape-web-service-sdk",
-	"version": "0.3.0",
+	"version": "0.3.1-develop.0",
```

Comment:
In the future, please submit hotfix changes on a `hotfix/` branch with merge target of `master`. 

For hotfixes, the version would just be incremented by a patch version without `-develop` at the end so it would be `0.3.1-develop.0`.

Don't worry about changing things around for this PR. I will merge to develop then deploy. Just wanted to inform you of how to submit one in the future.